### DISCUSSION TITLE
Standalone vs embedded with Next.js

### DISCUSSION THREAD

**Discussion by konradbjk** [sanity-io/sanity#5088] [Sanity Studio Support]
I want to use sanity for personal use this time. Potentially with a few projects (landing, blog, ...). Intuitively I believe installing it as a standalone and self hosting under my subdomain makes sense. Especially with custom components shared between projects - it looks like I can have multiple projects under one Sanity Studio (might be mistaken).

On the other hand I see most projects using a monorepo approach (i.e., using NextStudio).

What would be the best way to do it - embedded or a standalone?

In the end I plan to host a static pages via Cloudflare Pages (Next.js SSG), but I can change it to Vercel.

**Comment by piscopancer** [comment]
In your case, there is no need for a standalone app. The content is stored on the Sanity servers and you use your Next app to query it, it's that simple lol. So, I don't really understand why would one want a standalone sanity app.

Even more, you can go to sanity website and boot up a new project right there and then use its id in any front-end app.

Simply put, create a project on sanity website — use in multiple Next apps with embedded sanity admin dashboard, no standalone sanity app deployed anywhere.

**Reply by konradbjk** [reply]
Why potentially the standalone?

After tests. I can clearly see that Sanity seriously slows down my website on the first render. I do not mean the part on Studio, but the website itself.

Aside this. I work with people who have been using WordPress for 10+ years. They are used to being able to login into one place and manage multiple areas. So I give them one URL to login. They had a mixed feeling when I told them they need to do it project per project.
